src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedFieldFactory.js

```diff
@@ -49,13 +49,21 @@ export const FixedFieldFactory = {
         item.isArray = false;
       }
 
-      const value = content[item.code] || '';
+      const getValue = () => {
+        if (content[item.code]) {
+          return content[item.code];
+        }
+
+        return item.isArray
+          ? new Array(item.length).fill('\\')
+          : new Array(item.length + 1).join('\\');
+      };
 
       const itemSelect = (item.allowedValues)
         ? {
           type: item.isArray ? SUBFIELD_TYPES.SELECTS : SUBFIELD_TYPES.SELECT,
           allowedValues: item.allowedValues,
-          value,
+          value: getValue(),
```

Comment:
> In this case, form values are only relevant for dropdown lists others will be obsolete. Probably this function fillEmptyFixedFieldValues is what you need. And please cover this with tests.

It is still the same behavior. If you change the 6th position of the leader from `a` to `c`, the **form** value `Type` will remain `a`. Form values need to be always up to date.

Reply:
I don't see any issue with your last comment, my tests don't show any problems. Could U point my direction to change in Form? Probably defaultValue shoud be used not initialValue - in this is a problem?

Reply:
It is not okay to have out-of-date values in a form after changing any field on that form. We do not see any problems with this behavior now, but we might face them later and it will require a rethink of the entire approach. Here we already encountered outdated values and you've fixed it partially:
> @przemyslawturek - form values are not updated after changing the leader type. You change the view of row 008, but not the form values, and the values to send to BE are taken from form values.

I just don't want an issue like this to arise later.

You have updated the form state for the `Select` and `Selects` field types, but we have more types like: `Byte`, `String`, `Bytes` - these should also be up to date.
 
You can change the form values using the `useField` hook like this:
```
import { useField } from 'react-final-form';

const { input } = useField('records[2].content'); 
input.change(new008rowValues);
```

Video recording of the issue

https://github.com/folio-org/ui-quick-marc/assets/77053927/0eb8fbff-b691-4122-9ace-d00bc3a94c6b

Reply:
Ok the content of record with property "type" is not changing - I got it, but "type" is not part of Form for fixed field 008 - there is not form field Type in any document type specification 008.
So If U change the leader 06 and new value is provided then the change of 008 type (that not in specification) should be updated in LDR changing event.